Sources/SwiftInspectorAnalyzers/PropertyAnalyzer.swift

```diff
@@ -24,28 +24,15 @@ import Foundation
 import SwiftInspectorVisitors
 import SwiftSyntax
 
-// MARK: - PropertyAnalyzer
+extension StandardAnalyzer {
```

Comment:
rename file?

Reply:
I followed [this](https://github.com/fdiaz/SwiftInspector/blob/24e21504da3fb5f55e1d041f8085a2e123f56254/Sources/SwiftInspectorAnalyzers/ImportsAnalyzer.swift) example.